### ISSUE TITLE
Content Item Preview Toolbar - drop down handle for issues remains after the content is unselected

### ISSUE THREAD

**Issue by sgauruseu** [enonic/app-contentstudio#721]
1. Create a folder.
2. Select the folder and create 2 issues. Issues Menu appears on the preview toolbar (OK)
3. Unselect the content.

BUG: Drop down handle remains on the toolbar:

![image](https://user-images.githubusercontent.com/3728712/62529449-559c3480-b847-11e9-9356-27e7aebd712e.png)

**Comment by alansemenov** [comment]
This will be fixed once the workflow epic is merged into master.

**Comment by alansemenov** [comment]
Still relevant
